src/transfer/configuration_loader.py

```diff
@@ -76,40 +69,43 @@ def __resolve_endpoints(self, configs):
 
         dest_data_dir = configs["endpoints"]["dest_data_dir"]
         if dest_data_dir is None and re.match(
-            self.RegexPatterns.subject_datetime.value, raw_data_folder
+            EphysReaders.SourceRegexPatterns.subject_datetime.value,
+            raw_data_folder,
         ):
             configs["endpoints"]["dest_data_dir"] = (
                 "ecephys_" + raw_data_folder
             )
         if dest_data_dir is None and re.match(
-            self.RegexPatterns.ecephys_subject_datetime.value, raw_data_folder
+            EphysReaders.SourceRegexPatterns.ecephys_subject_datetime.value,
+            raw_data_folder,
         ):
             configs["endpoints"]["dest_data_dir"] = raw_data_folder
 
         if configs["endpoints"]["s3_prefix"] is None:
-            dest_data_folder = Path(
-                configs["endpoints"]["dest_data_dir"]
-            ).name
+            dest_data_folder = Path(configs["endpoints"]["dest_data_dir"]).name
             configs["endpoints"]["s3_prefix"] = dest_data_folder
 
         if configs["endpoints"]["gcp_prefix"] is None:
-            dest_data_folder = Path(
-                configs["endpoints"]["dest_data_dir"]
-            ).name
+            dest_data_folder = Path(configs["endpoints"]["dest_data_dir"]).name
             configs["endpoints"]["gcp_prefix"] = dest_data_folder
 
         if configs["register_on_codeocean_job"]["asset_name"] is None:
-            configs["register_on_codeocean_job"]["asset_name"] = (
-                configs["endpoints"]["s3_prefix"])
+            configs["register_on_codeocean_job"]["asset_name"] = configs[
+                "endpoints"
+            ]["s3_prefix"]
 
         if configs["register_on_codeocean_job"]["mount"] is None:
-            configs["register_on_codeocean_job"]["mount"] = (
-                configs["endpoints"]["s3_prefix"])
+            configs["register_on_codeocean_job"]["mount"] = configs[
+                "endpoints"
+            ]["s3_prefix"]
 
-        if (configs["jobs"]["register_to_codeocean"] and
-                configs["register_on_codeocean_job"]["api_token"] is None):
-            configs["register_on_codeocean_job"]["api_token"] = (
-                os.getenv('CODEOCEAN_API_TOKEN'))
+        if (
+            configs["jobs"]["register_to_codeocean"]
+            and configs["register_on_codeocean_job"]["api_token"] is None
+        ):
+            configs["register_on_codeocean_job"]["api_token"] = os.getenv(
+                "CODEOCEAN_API_TOKEN"
+            )
 
     def load_configs(self, sys_args):
```

Comment:
It would improve the code if we add the parameters, returns, etc. to the docstring

Reply:
@camilolaiton I can open a tech-debt issue to address this concern so that we can do the documentation and typing across the whole project. It's somewhat outside the scope of this task.

Reply:
Let me know if this ticket covers the tasks needed to be done: https://github.com/AllenNeuralDynamics/nd-data-transfer/issues/56